cpp/stdlib.h

```diff
@@ -107,6 +107,32 @@ inline void write(int fd, Str* s) {
   }
 }
 
+inline void setpgid(pid_t pid, pid_t pgid) {
+  pid_t ret = ::setpgid(pid, pgid);
+  if (ret < 0) {
+    throw Alloc<OSError>(errno);
+  }
+}
+
+inline int getpgid(pid_t pid) {
+  return ::getpgid(pid);
+}
+
+inline void tcsetpgrp(int fd, pid_t pgid) {
+  pid_t ret = ::tcsetpgrp(fd, pgid);
+  if (ret < 0) {
+    throw Alloc<OSError>(errno);
+  }
+}
+
+inline int tcgetpgrp(int fd) {
+  pid_t ret = ::tcgetpgrp(fd);
+  if (ret < 0) {
+    throw Alloc<OSError>(errno);
+  }
+  return ret;
+}
```

Comment:
should we unit test these? :thinking:

Reply:
I think it's OK to leave them off at first ... In some tests I have `for_test_coverage` that does trivial tests